src/clog/TraceEmitterModules/CLogSTDOUT.cs

```diff
@@ -302,16 +174,16 @@ public void TraceLineDiscovered(string sourceFile, CLogOutputInfo outputInfo, CL
                 switch (arg.TypeNode.EncodingType)
                 {
                     case CLogEncodingType.Int32:
-                        cast = "(int)";
+                        //cast = "(int)";
```

Comment:
Why are all these commented out?

Reply:
the goal here is to give every opportunity to the compiler to detect casting issues within the code using CLOG;  the casts puts the compiler at a disadvantage.  The comment should be removed in time, but having it helps clean up code that pass incorrect types - could I keep these comments for a bit and evaluate cleaning the comment later?